### PR TITLE
[Intel MKL] Fixed a bug in mkl_conv2d constant filter caching which c…

### PR DESCRIPTION
**Pull request by cuixiaom**
…aused the filter output tensor not get updated

### CODE REVIEW

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 1]
```diff
@@ -792,7 +790,9 @@ class MklConvOp : public OpKernel {
         // Tensorflow format to MKL format by caching the filter when it is
         // converted for the first time. This cached filter can then be reused
         // in subsequent iterations.
-        if (is_filter_const_) {
+        bool do_cache_filter =
+            src_dims[MklDnnDims::Dim_N] > kSmallBatchSize ? true : false;
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: Nit: The expression already returns boolean.
```suggestion
            (src_dims[MklDnnDims::Dim_N] > kSmallBatchSize);
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the fix!

File: tensorflow/core/kernels/mkl_conv_ops.cc (hunk 1 above)
Comment (cuixiaom) [reply tensorflow/core/kernels/mkl_conv_ops.cc]: Modified! Thanks!

**Review by penpornk — APPROVED** [review]
Thank you again for the PR!

**Pull request merged**